src/restaurantSide/pages/auth/ForgotPasswordHost.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import {
+  Grid,
+  Box,
+  TextField,
+  Typography,
+  Button,
+  Snackbar,
+  Alert,
+  IconButton,
+} from '@mui/material';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import { LogoImg } from './styled';
+
+const ForgotPasswordHost = () => {
+  const [email, setEmail] = useState('');
+  const [notification, setNotification] = useState({
+    open: false,
+    severity: '',
+    message: '',
+  });
+
+  const handleForgotPassword = async () => {
+    if (!email) {
+      setNotification({
+        open: true,
+        severity: 'error',
+        message: 'Please enter your email address.',
+      });
+      return;
+    }
+
+    setNotification({
+      open: true,
+      severity: 'success',
+      message: 'Password reset email sent. Check your inbox.',
+    });
+  };
+
+  const handleCloseNotification = () => {
+    setNotification({ ...notification, open: false });
+  };
+
+  const [isFocused] = useState(false);
```

Comment:
You should move this one to the useState section

Reply:
Done